empiric-ui/src/components/Details/EntriesTable.tsx

```diff
@@ -85,7 +85,7 @@ const EntriesTable: React.FC<EntriesTableProps> = ({
         accessorFn: (entry) => secondsToTime(entry.timestamp),
       },
     ],
-    [decimals, assetKey]
+    [decimals, currencyAlt, currencySrc]
```

Comment:
was this a bug?

Reply:
It appeared as a warning when building after we removed one of the columns from the table.